core/examples/luigi-sample-angular/src/luigi-config/extended/auth.js

```diff
@@ -104,6 +106,13 @@ class Auth {
     onAuthError: (settings, err) => {
       console.log('authErrorHandler 1', err, settings);
       // return false; // prevent redirect to logoutUrl, but go to /
+    },
+    onAuthExpireSoon: settings => {
+      console.log('onAuthExpireSoon ', settings);
+      Luigi.showAlert({
```

Comment:
Should the warning be displayed by Luigi itself and configurable through Luigi Config, and not configured by each application using Luigi?

Reply:
Clarified with @hardl, we are enabling already every Luigi application to easily provide custom handling for this event, no further actions are required.